API.md

```diff
@@ -401,6 +401,55 @@ After first iteration:
 */
 ```
 
+## `client.queryStorageDeal`
+
+> Query a storage deal's status
+
+### `client.queryStorageDeal(cid, [options])`
+
+#### Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| cid | `String` | CID of deal to query |
+| options | `Object` | Optional options |
+| options.signal | [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) | A signal that can be used to abort the request |
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Promise<Object>` | Storage deal |
+
+#### Example
+
+```js
+const dealcid = 'zDPWYqFD8CNXu7Mo9qPSUANbTK2vi9vJBnvavF9S1pVGPHafVHpT'
+const storageDeal = await fc.client.queryStorageDeal(dealcid)
+console.log(storageDeal)
+
+/*
+{
+  "State":7,
+  "Message":"",
+  "ProposalCid":
+    {
+      "/":"zDPWYqFD8CNXu7Mo9qPSUANbTK2vi9vJBnvavF9S1pVGPHafVHpT"
```

Comment:
This is an artifact of CID encoding in IPLD, to make it easier to work with we replace CID objects like this with the CID string. e.g.

https://github.com/filecoin-project/js-filecoin-api-client/blob/2c0e82fd8cfe7b959b1eb067a5d8e9960d2a16fe/src/cmd/actor/ls.js#L14